windows/runner/utils.h

```diff
@@ -0,0 +1,19 @@
+#ifndef RUNNER_UTILS_H_
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase